policyengine_us/variables/household/income/person/general/state_disability_insurance.py

```diff
@@ -0,0 +1,9 @@
+from policyengine_us.model_api import *
+
+
+class state_disability_insurance(Variable):
+    value_type = float
+    entity = Person
+    label = "State diability insurance (SDI)"
```

Comment:
I'm not sure this is in every state though. California defines its own (https://edd.ca.gov/disability), so I think we should move this into the California variable folder structure under `edd`. If you can determine that it's available similarly in other states we can keep here.
```suggestion
    label = "State disability insurance (SDI)"
```